paper-item.html

```diff
@@ -76,8 +78,14 @@
       is: 'paper-item',
 
       hostAttributes: {
-        role: 'listitem'
-      }
+        role: 'listitem',
+        tabindex: '0'
```

Comment:
i think now you can also fix the demo where i had to manually add `tabindex=0` to all the `paper-items`. Also, do you need it to `paper-icon-item` as well?